<launch>
    <node pkg="prizm_control" type="gostop_node" name="go_stop" output="screen" />
</launch>
